exercises/protein-translation/example.go

```diff
@@ -0,0 +1,65 @@
+package protein
+
+const testVersion = 1
+
+func proteinLookUp(codon string) string {
+	switch codon {
+	case
+		"AUG":
+		return "Methionine"
+	case
+		"UUU",
+		"UUC":
+		return "Phenylalanine"
+	case
+		"UUA",
+		"UUG":
+		return "Leucine"
+	case
+		"UCU",
+		"UCC",
+		"UCA",
+		"UCG":
+		return "Serine"
+	case
+		"UAU",
+		"UAC":
+		return "Tyrosine"
+	case
+		"UGU",
+		"UGC":
+		return "Cysteine"
+	case
+		"UGG":
+		return "Tryptophan"
+	case
+		"UAA",
+		"UAG",
+		"UGA":
+		return "STOP"
+	}
+	return "STOP"
```

Comment:
So I guess we haven't specified what should happen when the sequence is one not in the table, such as `AAA`, whether it should be an `error`. I'm OK with that, we can revisit if students ask for it.